lib/util/git/index.spec.ts

```diff
@@ -1,5 +1,6 @@
 import fs from 'fs-extra';
 import Git from 'simple-git';
+import SimpleGit from 'simple-git/src/git';
```

Comment:
```suggestion
import SimpleGit from 'simple-git';
```

I think a wrong auto import 😉

Reply:
This one is actually correct, because their default export is a factory function that returns an instance of the actual class. So to be able to partially mock it, I have to import the actual class. The alternative is to mock the factory function, but would probably look a bit hackier 😛. Open to naming the import something else if you want though!

Reply:
OK, didn't know that. So leave as is.